www/src/components/PrivateRoute.js

```diff
@@ -0,0 +1,8 @@
+const PrivateRoute = ({component: Component, authenticated, ...rest}) => {
+  return (
+    <Route {...rest} render={(props) => authenticated === true
+        ? <Component {...props} />
+        : <Redirect from={`${props.match.path}`} to={`/login`} />}
```

Comment:
Might need to look at this again because of the Switch route.

Example taken from [here](https://github.com/tylermcginnis/react-router-firebase-auth)